Example/Example/ContentView.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+import GEOSwift
+
+struct ContentView: View {
+    @State private var modifiableGeometry = GeometryModification()
+    @State private var tempGeometry = try! Geometry(wkt: "POLYGON((35 10, 45 45.5, 15 40, 10 20, 35 10),(20 30, 35 35, 30 20, 20 30))")
+    var body: some View {
+        VStack {
+            Text("Polygon")
+                .font(.title)
+                .padding()
+            GeometryReader { geometry in
```

Comment:
The app seems to work fine without this `GeometryReader`. I'm new to Swift UI though — was it needed for some reason?